source/slang/core.meta.slang

```diff
@@ -342,11 +342,18 @@ ${{{{
 // Declare built-in pointer type
 // (eventually we can have the traditional syntax sugar for this)
 }}}}
+
+__magic_type(NullPtrType)
+struct NullPtr
+{
+};
+
```

Comment:
This is fine for now, but we should have a longer-term design discussion around how null should ideally be handled in Slang.